src/js/task/crud.js

```diff
@@ -0,0 +1,136 @@
+/** @typedef { import('./index.js').Task } Task */
+
+const STORAGE_KEY = 'byteboard_tasks';
```

Comment:
just a note for the local storage, I think one thing to keep in mind is to make sure that each user has their own local store file. I know this key is for the tasks but we should consider using something like part of the user's name or username to append to the storage so that we know whose data is whose

Reply:
I don't think local storage has been implemented yet so this is just a future consideration

Reply:
This is a good point; one json file per user would probably be good.

Reply:
True. I haven't thought about that. I will make the key be `byteboard_tasks_${user_id}` later in the future.